src/remoteConnector.ts

```diff
@@ -684,6 +684,8 @@ export class RemoteConnector extends Disposable {
 				location: vscode.ProgressLocation.Notification
 			},
 			async () => {
+				// TODO(telemetry): it should be after connecting analytics event
```

Comment:
@mustard-mh it is not done yet

Reply:
I'm not clear about it. Right now we fallback to use local-app if `copy file` or `write ssh config` failed, but if we move it  inside, then we will not have fallback just failed? @akosyakov

Reply:
I don't want to change the logic only make sure that connecting is reported before and followed either by failed or success.

Reply:
We can also another event specific to local ssh configuration flow. `vscode_desktop_local_ssh_config` which is fired once either with success or failure, and then we can have failure/total ratio.

Reply:
> I don't want to change the logic only make sure that connecting is reported before and followed either by failed or success.

We can't do it, since this logic is way we can identify what we are going to connect with: `local-app` or `local-ssh`, otherwise, we will have two connecting event once fallback

> We can also another event specific to local ssh configuration flow. vscode_desktop_local_ssh_config which is fired once either with success or failure, and then we can have failure/total ratio.

Another event sounds good